ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -1612,9 +1765,9 @@ class RangeSelector {
 
                 // Skip animation
                 inputGroup.placed = chart.hasLoaded;
+                this.handleCollision(xOffsetForExportButton);
```

Comment:
Incorrect place. Buttons can be aligned to the right (eg rtl languages) and collision detection should include them no matter if inputs are enabled

Reply:
But this method is supposed to handle collision between the button group and input group only.